build/Targets/Settings.props

```diff
@@ -120,7 +120,7 @@
 
   <!-- Windows specific settings -->
   <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <DeployExtension Condition="'$(VisualStudioVersion)' != '15.0'">false</DeployExtension>
+    <DeployExtension Condition="'$(VisualStudioVersion)' != '15.0' and '$(VisualStudioVersion)' != '16.0'">false</DeployExtension>
```

Comment:
:bulb: This condition is always false. May as well remove it.

Reply:
Currently, yes. Until, we will start with version 17. I think it is done for the purpose.

---
In reply to: [217204514](https://github.com/dotnet/roslyn/pull/29016#discussion_r217204514) [](ancestors = 217204514)